ciphers/Caesar.java

```diff
@@ -24,8 +24,8 @@
     public static String encode(String message, int shift) {
         String encoded = "";
 
-        while (shift >= 26) { // 26 = number of latin letters
-            shift -= 26;
+        if (shift >= 26) { // 26 = number of latin letters
```

Comment:
Good catch! In fact, even this `if` is not necessary, since the modulo operation is no-op if `shift` < 26, i.e.

`shift % 26 == shift` if 0 <= `shift` < 26

Reply:
Oh! That's true! Less code then :)